includes/class-sensei-question.php

```diff
@@ -411,8 +411,13 @@ public function get_question_grade( $question_id ) {
      * @param $question_type
      */
     public static function load_question_template( $question_type ){
+		$template_loaded = Sensei_Templates::get_template( 'single-quiz/question_type-' . $question_type . '.php' );
 
-        Sensei_Templates::get_template  ( 'single-quiz/question_type-' . $question_type . '.php' );
+		if ( ! $template_loaded ) {
```

Comment:
I don't understand the logic here. We're already calling this function with this parameter when setting `$template_loaded`. Why would we call it again if the first call didn't work? I feel like I must be misreading the code or something.